app/views/runs/compare.slim

```diff
@@ -10,14 +10,9 @@
   h5
```

Comment:
The `short_run` variable at the top doesn't seem to be used anymore, so it can be deleted.